.github/issue_template.md

```diff
@@ -0,0 +1,11 @@
+## Problem
+<!--Add a description and the userstory prompt-->
+
+## Solution
```

Comment:
Would it make more sense if we didn't have a solution on the issue template? My reasoning is that there's a chance that implementation ends up differently than initially thought.

Reply:
I realized that this could also be a feature and show us how our idea could've changed/developed.